pkgs/top-level/all-packages.nix

```diff
@@ -22746,7 +22748,7 @@ in
 
   phabricator = callPackage ../misc/phabricator { };
 
-  physlock = callPackage ../misc/screensavers/physlock { };
+  physlock = callPackage ../misc/screensavers/physlock { }; 
```

Comment:
Trailing whitespace here as well.

Reply:
I did clean the history, but I'm not sure I did it correctly (I'm not used to squash commits...). Can you confirm that everything is good ?